tests/conformance/state/cosmosdb_test.go

```diff
@@ -0,0 +1,49 @@
+// +build conf
+
+package state
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	testcosmosdb "github.com/dapr/components-contrib/tests/conformance/state/cosmosdb"
+)
+
+// TODO remove cosmosdb specific test #599
```

Comment:
method to be removed as part of #599

Reply:
Can we just not include in the PR? We can add some more code as part of #599, and not have to check in something then immediately remove.

Reply:
will remove it.